src/Query.re

```diff
@@ -59,6 +60,7 @@ module Make = (Config: Config) => {
       | _ => NoData
       },
       result,
+      jsResult##refetch,
```

Comment:
actually I would just add this in the result type, to not extend the type, otherwise all simple usages get really verbose